### DISCUSSION TITLE
After upgrading to php 8.2.12 connection error unsafe legacy renegotiation disabled

### DISCUSSION THREAD

**Discussion by epocci** [PHPMailer/PHPMailer#3069] [Q&A]
Hi,
I updated my application to php 8.2.12 with OpenSsl 3.1.3, when I connect to a specific smtp server I get the error:
SSL routines::unsafe legacy renegotiation disabled

I've already set:

$mail->SMTPOptions = array(
        'ssl' => array(
        'verify_peer' => false,
        'verify_peer_name' => false,
        'allow_self_signed' => true
    ),
'tls' => array(
        'verify_peer' => false,
        'verify_peer_name' => false,
        'allow_self_signed' => true
    )
);  

Does anyone have the same error? How can I solve it?

**Comment by Synchro** [comment]
It means that your mail server is outdated. See approaches like [these](https://stackoverflow.com/questions/71603314/ssl-error-unsafe-legacy-renegotiation-disabled). You should not be disabling certificate verification, but you can probably use other configuration options in PHP to avoid the problem, however, this is a PHP config problem, not a PHPMailer issue.
